src/pedal/config.rs

```diff
@@ -0,0 +1,69 @@
+use crate::{pedal, Pedal as PedalTrait, Result};
+use cpal::StreamConfig;
+use serde::Deserialize;
+use Pedal::*;
+
+#[derive(Debug, Deserialize)]
+pub struct Config {
+    pedals: Vec<Pedal>,
```

Comment:
Ya this is confusing (per above naming comment), this is a vector of "pedal configs" right?